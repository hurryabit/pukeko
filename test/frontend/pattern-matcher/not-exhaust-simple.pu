data Choice a b = First a | Second b
f :: Choice a b -> a
f xy =
  case xy of
    First x -> x
