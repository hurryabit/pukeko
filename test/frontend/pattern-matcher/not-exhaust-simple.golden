test/frontend/pattern-matcher/not-exhaust-simple.pu:4:3: unmatched constructor: Second
